easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,29 @@
+name = 'MRtrix'
+version = '3.0_RC4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.brain.org.au/software/index.html#mrtrix'
+description = """MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner
+ robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/MRtrix3/mrtrix3/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['4f246e4cc6ddd5d2480d368342bcbeb9c5d21c04a3a4b5e6e2adb8c036611426']
+
+builddependencies = [
+    ('Eigen', '3.3.7', '', True),
+    ('pkg-config', '0.29.2'),
+]
+dependencies = [
+    ('zlib', '1.2.11'),
```

Comment:
@boegel `zlib` is already a dep of foss, so can it be removed?

Reply:
@smoors `zlib` is only an indirect dep of `foss`, so it's better to leave it in as an explicit dependency imho...